Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -1153,48 +1145,45 @@ void CSequenceBatchRenderDialog::OnUpdateEnd(IAnimSequence* sequence)
         cvarDebugInfo->Set(m_renderContext.cvarDisplayInfoBU);
     }
 
-    // Restore flags, range and the active director of the sequence.
-    sequence->SetFlags(m_renderContext.flagBU);
-    sequence->SetTimeRange(m_renderContext.rangeBU);
-    sequence->SetActiveDirector(m_renderContext.pActiveDirectorBU);
 
-    const auto imageFormat = m_ui->m_imageFormatCombo->currentText();
 
-    SRenderItem renderItem = m_renderItems[m_renderContext.currentItemIndex];
+    const SRenderItem& renderItem = m_renderItems[m_renderContext.currentItemIndex];
     if (m_bFFMPEGCommandAvailable && renderItem.bCreateVideo)
     {
         // Create a video using the ffmpeg plug-in from captured images.
         m_renderContext.processingFFMPEG = true;
 
-        AZStd::string outputFolder = m_renderContext.captureOptions.folder;
-        auto future = QtConcurrent::run(
-            [renderItem, outputFolder, imageFormat]
-        {
-            AZStd::string outputFile;
-            AzFramework::StringFunc::Path::Join(outputFolder.c_str(), renderItem.prefix.toUtf8().data(), outputFile);
+        const AZStd::string outputFolder = m_renderContext.captureOptions.folder;
+        const AZStd::string imageFormat = m_ui->m_imageFormatCombo->currentText().toStdString().c_str();
+        AZStd::string outputFile;
+        AzFramework::StringFunc::Path::Join(outputFolder.c_str(), renderItem.prefix.toUtf8().data(), outputFile);
 
-            QString inputFile = outputFile.c_str();
-            outputFile += ".webm";
+        // Create the input file string, leave the %06d unexpanded for the ffmpeg tool.
+        const AZStd::string inputFile = outputFile + "_%06d." + imageFormat;
 
-            // Use a placeholder for the input file, will expand it with replace.
-            QString inputFileDefine = "__input_file__";
+        outputFile += ".webm";
 
-            QString command = QStringLiteral("plugin.ffmpeg_encode '%1' '%2' '%3' %4 %5 'crop=%6:%7:0:0'")
-                .arg(inputFileDefine).arg(outputFile.c_str()).arg("libvpx-vp9")
-                .arg(10240).arg(renderItem.fps).arg(getResWidth(renderItem.resW)).arg(getResHeight(renderItem.resH));
+        static const char* filePattern = "__input_file__";
 
-            // Create the input file string, leave the %06d unexpanded for the mpeg tool.
-            inputFile += "_%06d.";
-            inputFile += imageFormat;
+        QString command = AZStd::string::format(
+            "plugin.ffmpeg_encode '%s' '%s' '%s' %i %i 'crop=%i:%i:0:0'",
+            filePattern,
+            outputFile.c_str(),
+            "libvpx-vp9",
+            10240,
+            renderItem.fps,
+            getResWidth(renderItem.resW),
+            getResHeight(renderItem.resH)).c_str();
 
-            // Replace the input file
-            command = command.replace(inputFileDefine, inputFile);
+        command = command.replace(filePattern, inputFile.c_str());
 
+        auto future = QtConcurrent::run([command]
```

Comment:
hmmm, I know it was already there but I find it weird to use the QT threading implementation instead of the engine one, its typically something that you want you engine to have control over